examples/resources/hpegl_vmaas_load_balancer_pool/nsx_t_lb_pool.tf

```diff
@@ -1,6 +1,7 @@
 # (C) Copyright 2022 Hewlett Packard Enterprise Development LP
 
 resource "hpegl_vmaas_load_balancer_pool" "test_lb_pool" {
+  lb_id = data.hpegl_vmaas_load_balancer.lb.id 
```

Comment:
How can the user get these two values ?
 passive_monitor_path = 136
active_monitor_paths = 133

IN UI its a dropdown with some string values.
![image](https://user-images.githubusercontent.com/78580773/177915151-66adeb93-f1e6-4cc8-ab8e-625ddc48dde0.png)

Reply:
I also have a comment on the resource definition here. I assume it should be similar to UI so that its easy for end user in terms of End Usage.
![image](https://user-images.githubusercontent.com/78580773/177915307-d88afdcc-d2bf-4188-8afb-2929416eb3af.png)

@gandharvas  Need your input here

Reply:
vip_balance = "ROUND_ROBIN" should have naming similar to UI

Reply:
snat_ip_address = "" field in the UI. do we need this input from user ?

Reply:
member_group  is dropdown in UI. why so many fields from user here ?

Reply:
I see tags and scope in UI, don't we need them here ?

Reply:
> I also have a comment on the resource definition here. I assume it should be similar to UI so that its easy for end user in terms of End Usage. ![image](https://user-images.githubusercontent.com/78580773/177915307-d88afdcc-d2bf-4188-8afb-2929416eb3af.png)
> 
> @gandharvas Need your input here

I totally agree with you Manju, Any deviation from UI fields should be strictly restricted to mandatory fields only.(Which we cannot auto-populate)